packages/linter/src/generators/utils/eslint-file.ts

```diff
@@ -0,0 +1,22 @@
+import type { Tree } from '@nrwl/devkit';
+
+const eslintFileList = ['.eslintrc.json', '.eslintrc.js'];
+
+export function containsEslint(tree: Tree): boolean {
+  for (const file of eslintFileList) {
+    if (tree.exists(file)) {
+      return true;
+    }
+  }
+  return false;
+}
+
+export function findEslintFile(tree: Tree): string {
+  for (const file of eslintFileList) {
+    if (tree.exists(file)) {
+      return file;
+    }
+  }
+  // Default file
+  return 'eslintrc.json';
```

Comment:
@AloisH @FrozenPandaz I know this has already been merged, but why wasn't this `.eslintrc.json`?

It has changed the behaviour inside of `packages/linter/src/generators/lint-project/lint-project.ts` above

`eslintrc.json` without a leading dot is not ever referenced by https://eslint.org/docs/user-guide/configuring/configuration-files.

This seems to be intentional because you have multiple unit tests asserting this, but I haven't understood why we would want to make this change?

Reply:
Hello @JamesHenry ! I think I made a mistake ! Thank you for noticing it !
I got one test trying it but I think I just forgot to put a dot.

Should we revert this PR so I can make a new commit fixing it ?